src/Agent/NewRelic/Agent/Core/Logging/LoggerBootstrapper.cs

```diff
@@ -85,7 +84,7 @@ public static void Initialize()
             GlobalContext.Properties["pid"] = new ProcessStatic().GetCurrentProcess().Id;
 
             SetupStartupLogAppender(logger);
-            SetupConsoleLogAppender(logger);
+            SetupConsoleLogAppender(logger, Level.Warn);
```

Comment:
After thinking about the use case (containerized process with no FS), I think if the ConsoleLog config is enabled then we should log at the specified config level from the start. If we only log warn events in this case, we could miss some startup/initialization messages of interest before the switch happens.

Reply:
Oh, I see. This is a console logger that is setup before we even read the logging config.

Reply:
I was going for minimal changes to existing agent behavior.  If the console logger is enabled in the agent config, the level will get updated to the configured level as soon as the config is parsed, which happens very early on in the startup process.

I'm assuming that whoever wanted the initial console logger threshold set to WARN had a good reason to do so. 🤷‍♂️